README.rst

```diff
@@ -12,13 +12,31 @@ Sequence Independent Molecular Replacement Based on Available Database
 
 .. image:: https://img.shields.io/badge/solution%20count-4-blue.svg?style=flat
    :target: http://www.rcsb.org/pdb/results/results.do?tabtoshow=Current&qrid=C03A7140
-   :alt: Solution count 
+   :alt: Solution count
 
-Documentation & Usage
-+++++++++++++++++++++
-* Please refer to `SIMBAD's installation page <http://simbad.readthedocs.io/en/latest/install.html>`_ for information on how to install SIMBAD and to maintain SIMBAD databases.
-* Please refer to `SIMBAD's examples page <http://simbad.readthedocs.io/en/latest/examples.html>`_ to see examples of how to run SIMBAD.
-* Please refer to `SIMBAD's documentation page <http://simbad.readthedocs.io/en/latest/contents.html>`_ for additional command line options and for API documentation.
+About
++++++
+
+SIMBAD is a sequence independant molecular replacement pipeline developed by the Rigden Lab at the University of Liverpool.
```

Comment:
Maybe not use "Rigden Lab", I think Dan is not too keen on that term. Maybe write "by the group of `Daniel Rigden <https://www.liverpool.ac.uk/integrative-biology/staff/daniel-rigden/>_ at the ..."